examples/openai/README.md

```diff
@@ -1,7 +1,298 @@
-# OpenAI Example
-1. Rename .env.local to .env and add your OpenAI API key to it.
+# OpenAI plugin examples
+
+This module provides examples of usage of OpenAI plugin for Genkit, which wraps around the
+OpenAI API.
+## Set up Genkit packages and DevUI
+1. Rename .env.local to .env and add your OpenAI API key to it, or specify it in the environment variable `OPENAI_API_KEY`.
 2. Run `npm install` to install the dependencies.
-3. Run `npm run start` to launch the Genkit Dev UI.
+3. Run `npx genkit start` to launch the Genkit Dev UI.
+
+Genkit is configured from the `index.ts`, where you can import and initialize the plugin and define prompts, flows, models and other tools which could be accessed directly through Gekit Dev UI:
+
+```
+import { configureGenkit } from '@genkit-ai/core';
+import { openAI } from 'genkitx-openai-plugin';
+
+export default configureGenkit({
+  plugins: [
+    openAI({ apiKey: process.env.OPENAI_API_KEY }),
```

Comment:
for conciseness we can even skip this as the plugin will fetch it from this env variable automatically. So we can just put openAI()

Reply:
I'll check, but yes,it should work automatically